pom.xml

```diff
@@ -409,6 +409,26 @@
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-checkstyle-plugin</artifactId>
             <version>3.4.0</version>
+           <exclusions>
```

Comment:
Indentation is a bit off.

Reply:
I updated the indentation on this line. It was missing a space to line up with the lines above.